### PR TITLE
Add unittest for infernce

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

**Comment by dreamerlin** [comment]
It seems something went wrong in github CI, can u help rerun it? @innerlee

**Comment by innerlee** [comment]
image

mem or cpu exceeded quota?

**Comment by innerlee** [comment]
Could you try again with a tiny model?

**Comment by dreamerlin** [comment]
> Could you try again with a tiny model?

Resolve it by using CenterCrop for inferecing instead of TenCrop.

File: tests/test_inference.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+def test_inference_recognizer():
+    if torch.cuda.is_available():
+        device = 'cuda:0'
+    else:
+        device = 'cpu'
+    model = init_recognizer(config_file, None, device)
+
+    for ops in model.cfg.data.test.pipeline:
+        if ops['type'] == 'TenCrop':
+            # Use CenterCrop to reduce memory
```
Comment (innerlee) [inline tests/test_inference.py:44]: Use CenterCrop to reduce memory in order to pass CI

**Pull request merged**